supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc

```diff
@@ -122,13 +155,15 @@
 [discrete]
 [[trusted-certificate-authority]]
 ==== image:images/yes.png[yes] trusted Certificate Authority (noun)
-*Description*: A "trusted Certificate Authority" is a third party that creates SSL certificates (CA certificates) used for authentication. It is not to be confused with self-signed certificates. Note the capitalization of Certificate Authority, commonly abbreviated as CA.
+*Description*: A trusted Certificate Authority (CA) is a third-party entity that creates TLS certificates, known as CA certificates, for authentication purposes. A trusted CA is different from a self-signed certificate in that a self-signed certificate has its own private key and does not need to request a key from a public or private CA. Note the capitalization of Certificate Authority, which is commonly abbreviated as CA.
```

Comment:
You know, as soon as I saw this entry, I was going to say that I doubted that it should be capitalized "Certificate Authority". I see that it was already like that though. However, I don't think it should be capitalized - we should check this (and can do so in a separate PR though)